versions/3.1.0.md

```diff
@@ -3264,6 +3264,7 @@ Field Name | Type | Description
 <a name="oauthFlowsPassword"></a>password| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Resource Owner Password flow 
 <a name="oauthFlowsClientCredentials"></a>clientCredentials| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Client Credentials flow.  Previously called `application` in OpenAPI 2.0.
 <a name="oauthFlowsAuthorizationCode"></a>authorizationCode| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Authorization Code flow.  Previously called `accessCode` in OpenAPI 2.0.
+<a name="oauthFlowsDeviceCode"></a>deviceCode| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Device Code flow.
```

Comment:
I called it "Device Code" because that is how <https://oauth.net/2/grant-types/device-code> names it. However, the RFC is named "OAuth 2.0 Device Authorization Grant". Should I change this to be named "Device Authorization"?